src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/#Trim.txt#

```diff
@@ -0,0 +1,77 @@
+// ******** STRING PARAMETERS ********
```

Comment:
why the # in this file's name?

Reply:
It's a long story, but this was a mistake and the file has been removed.  It was a copy of Trim.txt.